lib/jobs/install-os.js

```diff
@@ -96,6 +96,23 @@ function installOsJobFactory(
                 }
             });
         }
+
+        if (this.options.installPartitions) {
+            _.forEach(this.options.installPartitions, function(partition) {
+                assert.string(partition.mountPoint);
+                assert.string(partition.size);
```

Comment:
I mean if the `size` is omitted, then you can default to auto mechanism, normally it is a number.

Of course, it is OK to explicitly support a "auto" string, but I think at least the `number` type should be support, because it is more straightforward.